conans/test/functional/generators/cmake_find_package_multi_test.py

```diff
@@ -76,3 +77,55 @@ def test_native_export_multi(self):
                         self.assertIn("Hello World {}!".format(bt), c.out)
                         self.assertIn("bye World {}!".format(bt), c.out)
                         os.remove(os.path.join(c.current_folder, "example"))
+
+    def cpp_info_name_test(self):
+        client = TestClient()
+        client.run("new hello/1.0 -s")
+        replace_in_file(os.path.join(client.current_folder, "conanfile.py"),
+                        'self.cpp_info.libs = ["hello"]',
+                        'self.cpp_info.libs = ["hello"]\n        self.cpp_info.name = "MYHELLO"',
+                        output=client.out)
+        client.run("create .")
+        client.run("new hello2/1.0 -s")
+        replace_in_file(os.path.join(client.current_folder, "conanfile.py"),
+                        'self.cpp_info.libs = ["hello"]',
+                        'self.cpp_info.libs = ["hello"]\n        self.cpp_info.name = "MYHELLO2"',
+                        output=client.out)
+        replace_in_file(os.path.join(client.current_folder, "conanfile.py"),
+                        'exports_sources = "src/*"',
+                        'exports_sources = "src/*"\n    requires = "hello/1.0"',
+                        output=client.out)
+        client.run("create .")
+        cmakelists = """
```

Comment:
No need to change it now, we need to do the release. But for the future, please try to use ``textwrap.dedent()`` for cleaner tests layout.